spec/support/spec_setup.rb

```diff
@@ -0,0 +1,71 @@
+require_relative './spec_config'
+
+class SpecSetup
+  def run
+    begin
+      create_user(admin_unauthorized_client, SpecConfig.instance.root_user)
+    rescue Mongo::Error::OperationFailure => e
```

Comment:
This is awesome! Throwing away all the errors like we were doing before when creating users was definitely not a good solution; checking them to make sure we don't throw away meaningful errors is super helpful